pyblish/logic.py

```diff
@@ -236,6 +238,37 @@ def plugins_by_host(plugins, host):
     return compatible
 
 
+def plugins_by_targets(plugins):
```

Comment:
Let's pass in targets ourselves, so it resembles the other `by_something()` functions.

`def plugins_by_targets(plugins, targets)`